app/controllers/admin/redirections_controller.rb

```diff
@@ -1,6 +1,9 @@
 class Admin::RedirectionsController < AdminController
+  DEFAULT_SORT_PARAMS = {"sort_key" => "next_id", "sort_dir" => "desc"}
+
   def index
-    @redirections = Redirection.order(created_at: :desc)
+    order_by = [DEFAULT_SORT_PARAMS.merge(sort_params).values].to_h
```

Comment:
This is a little tricky. I had to paste it into `irb` to see what it was doing. How about this?

```ruby
sort_by = DEFAULT_SORT_PARAMS.merge(sort_params)
order_by = { sort_by["sort_key"] => sort_by["sort_dir"] }
```

Reply:
Oh yeah, that's much better.